### PR TITLE
make docs.ubports.com/porting a valid link

### PR DESCRIPTION
**Pull request by Fuseteam**

### CODE REVIEW

**Comment by doniks** [comment]
as far as I can *test* it, it looks ok to me, but I'm wondering about two things:

1. the capital I in Index. Is this going to work that http://docs.ubports.com/en/latest/porting will properly get redirected to this page? I think it should get redirected, but I'm wondering whether it will work with the capital I. In all other places in the docs, it's lower caps index.rst
2. and relatedly, if this redirecting would work properly, can't we then leave the word "Index" out completely? I mean, e.g., link to `/porting` instead of `/porting/Index` or `/porting/Index.rst` 

I can't find a way to properly test this, because in Jenkins, and when I build it locally, it simply shows me the directory listing, e.g., https://ci.ubports.com/job/docs.ubports.com/job/PR-373/1/artifact/_build/html/, but in the live website this url gets redirected to the index

Unless, I'm missing something, I would much prefer to see it renamed to lower caps index and then removed where it isn't needed.

**Comment by Fuseteam** [comment]
@doniks alright i'll make those changes then

**Comment by Fuseteam** [comment]
i noticed in systemdev/kernel-hal.rst /index is used for `appdev/index` and `systemdev/index`
should i still remove index from porting/index?

**Comment by Fuseteam** [comment]
before i forget removing `/index` from redirections.txt redirects to `porting` even if it's specified as `porting/`

**Comment by doniks** [comment]
> i noticed in /index.rst `/index` is used for `appdev/index` and `systemdev/index`
> should i still remove index from porting/index?

I see. Ok, let's keep it the way it is now



> before i forget removing `/index` from redirections.txt redirects to `porting` even if it's specified as `porting/`

ah. ok. good!

File: redirects.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
-"porting/introduction.rst" "porting/Intro.rst"
+"porting/introduction.rst" "porting/index.rst"
+"porting/Intro.rst" "porting/index.rst"
+"porting/Index.rst" "porting/index.rst"
```
Comment (doniks) [inline redirects.txt:3]: this one we don't need now, right? I mean before this PR, there was never a mention of "porting/Index.rst", so we don't need to redirect it.

**Review by doniks — CHANGES_REQUESTED** [review]

File: redirects.txt (hunk 1 above)
Comment (Fuseteam) [reply redirects.txt:3]: without this it won't build for me locally

Comment (peter-pycom) [reply redirects.txt:3]: strange. maybe after "make clean", like rm -rf _build or smt like that?

Comment (Fuseteam) [reply redirects.txt:3]: hmmm with a squash rebase it does build fine without it, should i just force push?

Comment (peter-pycom) [reply redirects.txt:3]: ok :+1:

Comment (Fuseteam) [reply redirects.txt:3]: > strange. maybe after "make clean", like rm -rf _build or smt like that?

oh interesting it appears related to _build

Comment (doniks) [reply redirects.txt]: Yeah, I've seen smt similar. I guess the build process doesn't delete files from the build folder

**Review by doniks — APPROVED** [review]

**Pull request merged**